### PR TITLE
Small updates to dome

### PR DESCRIPTION
**Pull request by wtgee**
More dome changes are on the way but I accidentally was working in my `develop` branch and didn't want to disentangle so am just submitting here. :)

These are the updates that existed on the Mac from the latest run.

### CODE REVIEW

**Review by bazkiaei — APPROVED** [review]

File: conf_files/huntsman.yaml [hunk 1]
```diff
@@ -30,14 +30,11 @@ directories:
     resources: POCS/resources/
     targets: /var/huntsman/huntsman-pocs/conf_files
     webcam: webcams
-cameras:
```
Comment (AnthonyHorton) [inline conf_files/huntsman.yaml:33]: Removing the camera section from the default config file seems to have broken the tests. Don't think it should, though.

**Review by AnthonyHorton — CHANGES_REQUESTED** [review]
Tidying of musca.py is fine, but need to fix the tests.

File: conf_files/huntsman.yaml (hunk 1 above)
Comment (wtgee) [reply conf_files/huntsman.yaml:33]: I'll get this fixed.  Issue here https://github.com/panoptes/POCS/issues/733

**Comment by wtgee** [comment]
Closing in favor of #106

**Pull request closed without merging**